docs/snippets/all/howto/any_batch_value_send_columns.py

```diff
@@ -17,8 +17,8 @@
     times=[rr.TimeSequenceColumn("step", timestamps)],
     components=[
         # log one value per timestamp
-        rr.AnyBatchValue("custom_component_single", one_per_timestamp),
+        rr.AnyBatchValue(rr.ComponentDescriptor("custom_component_single"), one_per_timestamp),
```

Comment:
Need to add some auto-upcasting magic, end-users shouldn't have to bother with this in the common/simple case.